frontend/src/v5/ui/routes/viewer/tickets/ticketDetails/ticketsDetailsCard.component.tsx

```diff
@@ -51,16 +56,22 @@ export const TicketDetailsCard = () => {
 	const defaultView = ticket?.properties?.[AdditionalProperties.DEFAULT_VIEW];
 	const currentIndex = filteredTickets.findIndex((tckt) => tckt._id === ticket._id);
 	const initialIndex = useRef(currentIndex);
+	const disableCycleButtons = currentIndex > -1 ? filteredTickets.length < 2 : filteredTickets.length < 1;
+
+	const getUpdatedIndex = (delta: number) => {
+		let index = currentIndex === -1 ? initialIndex.current : currentIndex;
```

Comment:
for this to work, you need to update the `initialIndex.current` everytime the ticket changes, like:
```ts
useEffect(() => {
	initialIndex.current = currentIndex;
}, [ticket?._id]);
```

Otherwise, reproduce this scenario:
- You have 4 tickets, all named "AAA";
- In the search input, type in "AAA";
- Open up the first ticket (initialIndex.current === 0);
- Using the arrows, reach the third ticket
- Change the name to "BBB"
- Using the arrows, go to the previous ticket

As a result, you will reach the fourth ticket as opposed to the second